db/controllers/controllers.js

```diff
@@ -48,8 +49,13 @@ exports.getCommentsByArticleId = (req, res, next)=>{
     })
 }
 exports.getAllArticles = (req,res,next)=>{
-    fetchAllArticles()
-    .then((articles)=>{
+   const {topic}=req.query
+
+    const promises=[fetchAllArticles(topic)]
```

Comment:
Not sure what we are doing here? We are doing a promise.all but just on one promise? We also have some commented out code below that will need removing if we aren't using it.

What test are we working on? There is no test in this commit?

Reply:
Okay, yes think I've fixed/amended all this